checkov/terraform/checks/resource/alicloud/PasswordPolicyMaxAge.py

```diff
@@ -0,0 +1,36 @@
+from checkov.common.models.enums import CheckResult, CheckCategories
+from checkov.terraform.checks.resource.base_resource_value_check import BaseResourceValueCheck
+from checkov.common.util.type_forcers import force_int
+
+
+class PasswordPolicyMaxAge(BaseResourceValueCheck):
+    def __init__(self):
+        name = "Ensure Ram Account Password Policy Max Age less than/equal to 90 days"
+        id = "CKV_ALI_24"
+        supported_resources = ['alicloud_ram_account_password_policy']
+        categories = [CheckCategories.IAM]
+        super().__init__(name=name, id=id, categories=categories, supported_resources=supported_resources)
+
+    def get_inspected_key(self):
+        return 'max_password_age'
+
+    def get_expected_value(self):
+        return 60
```

Comment:
```suggestion
        return 90
```
either `90` here or change everywhere else `90` -> `60` 😉